{{cookiecutter.project_slug}}/docs/howto.rst

```diff
@@ -4,20 +4,19 @@ How To - Project Documentation
 Get Started
 ----------------------------------------------------------------------
 
-Documentation can be written as rst files in the `{{cookiecutter.project_slug}}/docs/_source`.
+Documentation can be written as rst files in `{{cookiecutter.project_slug}}/docs`.
```

Comment:
I guess the intend of these `` ` `` is to have path formatted as `preformatted` not _italic_, which is done with double backtick ``` `` ``` in RST, single backtick work in Markdown... 

WDYT? Would you mind fixing them in this file?

Reply:
ah sorry I missed this comment! Yes, I agree. I can fix those in a separate pr.

Reply:
Yes, wan't directly related, so I decided to go ahead with it.